src/mlpack/methods/kernel_pca/kernel_rules/naive_method.hpp

```diff
@@ -77,7 +81,9 @@ class NaiveKernelRule
 
   // Swap the eigenvalues since they are ordered backwards (we need largest to
   // smallest).
-  for (size_t i = 0; i < floor(eigval.n_elem / 2.0); ++i)
+  omp_size_t varForOpenmp = floor(eigval.n_elem / 2.0);
+  #pragma omp parallel for
+  for (omp_size_t i = 0; i < varForOpenmp ; i++)
```

Comment:
Not necessarily this one, but consider renaming this variable
```suggestion
  omp_size_t num_eigenvals = floor(eigval.n_elem / 2.0);
  #pragma omp parallel for
  for (omp_size_t i = 0; i < num_eigenvals; i++)
```